### PR TITLE
Updates to sensu emr checks and metrics

### PR DESCRIPTION
**Pull request by babsher**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: bin/check-emr-steps.rb [hunk 1]
```diff
@@ -60,19 +72,19 @@ def run
...
       messages = []
       now = Time.new
       failed = steps.select { |step| now - step.status.timeline.end_date_time < 10 * 60 }
-      failed.each_entry { |step| messages << "Step #{step.id} '#{step.name}' has failed on cluster #{cluster.id} '#{cluster.name}'" }
-
-      if messages.count > 0
-        critical("#{messages.count} #{messages.count > 1 ? 'steps have' : 'step has'} failed: #{messages.join(',')}")
-      else
-        ok
+      if failed.size > config[count]
```
Comment (eheydrick) [inline bin/check-emr-steps.rb]: `count` should be a symbol

**Comment by eheydrick** [comment]
can you rebase against master

**Comment by babsher** [comment]
ya rebased

File: CHANGELOG.md [hunk 2]
```diff
@@ -148,8 +150,7 @@ WARNING:  This release contains major breaking changes that will impact all user
 ### Added
 - initial release
 
-[Unreleased]: https://github.com/sensu-plugins/sensu-plugins-aws/compare/2.3.0...HEAD
-[2.3.0]: https://github.com/sensu-plugins/sensu-plugins-aws/compare/v2.2.0...2.3.0
+[Unreleased]: https://github.com/sensu-plugins/sensu-plugins-aws/compare/2.2.0...HEAD
```
Comment (eheydrick) [inline CHANGELOG.md]: ? 2.3.0 should be left in.

Comment (babsher) [inline CHANGELOG.md]: fixed

**Comment by eheydrick** [comment]
Thanks @babsher!

**Pull request merged**